modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc

```diff
@@ -364,7 +364,7 @@ A cache is introduced to avoid unnecessary LDAP traffic, for example caching use
 Saving the configuration empties the cache. 
 The time is given in seconds.
 Note that almost every PHP request requires a new connection to the LDAP server. 
-If you require fresh PHP requests we recommend defining a minimum lifetime of 15s or so, rather than completely eliminating the cache.
+If you require fresh PHP requests, we recommend defining a minimum lifetime of about 15 secondsor higher, rather than completely eliminating the cache.
```

Comment:
```suggestion
If you require fresh PHP requests, we recommend defining a minimum lifetime of about 15 seconds or higher, rather than completely eliminating the cache.
```